hexrdgui/hexrd_config.py

```diff
@@ -940,22 +940,23 @@ def create_masked_images_dict(self, fill_value=0, display=False):
             # and no panel buffers.
             fill_value = 0
 
-        raw_masks_dict = self.create_raw_masks_dict(display=display)
+        raw_masks_dict = self.create_raw_masks_dict(images_dict,
+                                                    display=display)
         for det, mask in raw_masks_dict.items():
+            img = images_dict[det]
             if has_panel_buffers:
                 panel = instr.detectors[det]
                 utils.convert_panel_buffer_to_2d_array(panel)
 
-            for name, img in images_dict.items():
```

Comment:
@bnmajor I got rid of this extra loop, because it appears to be looping over detector names as well (same as the outer loop). Please let me know if I was incorrect to do so.

Reply:
I think that you are correct, this was redundant code that should have been removed in a previous refactor of this function... LGTM!